.eslintrc

```diff
@@ -65,7 +65,7 @@
     }],
     "max-nested-callbacks": ["error", 5],
     "max-params": ["error", 3],
-    "max-statements": ["error", 10, { "ignoreTopLevelFunctions": true }],
+    "max-statements": ["error", 34, { "ignoreTopLevelFunctions": true }],
```

Comment:
not sure this is the right place for it, i would put it only on the required places if needed...? @gailsteiger ?